test_app/views.py

```
@@ -288,3 +289,105 @@ def otel_logs(request):
+        'populate': get_fully_qualified_url('org-delete-populate', request=request),
+        'bare': get_fully_qualified_url('org-delete-bare', request=request),
+        'deferred': get_fully_qualified_url('org-delete-deferred', request=request),
+        'optimized': get_fully_qualified_url('org-delete-optimized', request=request),
+    })
+
+
+ORG_DELETE_PREFIX = 'scale-profile'
+
+
+@api_view(['GET'])
+def org_delete_populate(request, format=None):
```

Ruff (0.15.21)

[error] Function argument `format` is shadowing a Python builtin

(A002)